class-00/lab-c/starter-code/styles/base.css

```diff
@@ -0,0 +1,21 @@
+@import url('https://fonts.googleapis.com/css?family=Abel|Oswald:700|Yellowtail|Patrick+Hand');
+
+
+* {
+  font-family: times;
+}
+
+body {
+  background: #fff;
+}
+main p {
+  color: #333;
+  font-size: 1.2em;
+  line-height: 1.3em;
+  text-indent: 30px;
+  margin: 20px 0px;
+}
+
+input {
```

Comment:
Maybe this is a good place for this, I'm not sure. Can you explain to me why you decided to have the input font here instead of in another style sheet?